### DISCUSSION TITLE
Any chance of a patch release?

### DISCUSSION THREAD

**Discussion by naxxfish** [Rapptz/discord.py#8304] [General]
I'd like to use this module in a project but I have a transient dependency on aiohttp that is `<=3.8.1,>=3.8.0`.  The 1.7.3 release specifies `<3.8.0`

I can see on `master` we're asking for `<4` which would work fine but even `3.8.1` would work.

**Comment by HigherOrderLogic** [comment]
You can use virtual env to solve this.

**Comment by Rapptz** [comment]
Unfortunately I have no plans to make a version bump for the v1.x version. The v2 version of the library should hopefully be out relatively soon.
